pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -1039,8 +1015,15 @@ func GetBootstrapClusterRoleBindings() []rbac.ClusterRoleBinding {
 
 	finalClusterRoleBindings := []rbac.ClusterRoleBinding{}
 	finalClusterRoleBindings = append(finalClusterRoleBindings, openshiftClusterRoleBindings...)
-	finalClusterRoleBindings = append(finalClusterRoleBindings, kubeControllerClusterRoleBindings...)
 	finalClusterRoleBindings = append(finalClusterRoleBindings, openshiftControllerClusterRoleBindings...)
+
+	// add default metadata only to openshift cluster role bindings
+	for i := range finalClusterRoleBindings {
+		roleBinding := &finalClusterRoleBindings[i]
+		addDefaultMetadata(roleBinding)
```

Comment:
What is the use-case for `GetOpenshiftBootstrapClusterRoleBindings` to be missing annotations?